### PR TITLE
Adds and adjusts information files

### PR DESCRIPTION
**Pull request by ethan-davies**
## Summary
This Pull Request adds informative files to the respiratory the changes are as followed

- Created a `CODE_OF_CONDUCT.md`
- Created a `SECURITY.md` based of confirmed website features
- Created a `CONTRIBUTING.md`
- Updated the organisation `README.md` to link some of these information files

## Useful information
By adding these files to the `.github` repository will not mean it will get copied to the other repository’s source instead, with the `SECURITY.md` and `CODE_OF_CONDUCT.md` will get placed in a tab of there respective sections. **In short,** this means we would need a separate PR to add the files the the repository’s source to make it easily accessible to the visitor.

<sub>My bad for the unnecessary commits I’m currently on holiday and had to do it on mobile</sub>

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
-## 📂 Stay Updated
+## Stay Informed
 
-Stay up-to-date with the latest news and developments:
+Stay up-to-date with the latest news and progress:
 
-- Visit our website: [hybrismc.dev](https://hybrismc.dev)
-- Join our Discord server: [discord.gg/Hybris](https://discord.gg/Hybris) (that's where we stay in touch with the community)
+- Visit our official website: [hybrismc.dev](https://hybrismc.dev)
+- Follow us on Twitter: [@COMING-SOON](https://x.com/)
```
Comment (770grappenmaker) [inline profile/README.md]: This should not be there, we will not make a Twitter/X account.

File: profile/README.md [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
 
-## ⚖️ License
+## Licensing
 
-Most of Hybris is licensed under the [GNU General Public License v3.0](https://github.com/HybrisMC/.github/blob/main/LICENSE).
+Hybris is mostly licensed under the [GNU General Public License v3.0](../LICENSE).
```
Comment (770grappenmaker) [inline profile/README.md]: "Most of the projects maintained by Hybris" is probably better

**Review by 770grappenmaker — CHANGES_REQUESTED** [review]
The rest LGTM

**Comment by ethan-davies** [comment]
I have just resolved the requested changes.

**Review by 770grappenmaker — APPROVED** [review]
Will await @RichardDorian for review.

**Comment by RichardDorian** [comment]
Looks good to me too!

**Review by RichardDorian — APPROVED** [review]
Looks good to me too!

**Pull request merged**